src/libraries/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs

```diff
@@ -51,11 +51,14 @@ public void DumpRuntimeInformationToConsole()
 
             Console.WriteLine($"### CURRENT DIRECTORY: {Environment.CurrentDirectory}");
 
-            Console.WriteLine($"### CGROUPS VERSION: {Interop.cgroups.s_cgroupVersion}");
-            string cgroupsLocation = Interop.cgroups.s_cgroupMemoryLimitPath;
-            if (cgroupsLocation != null)
+            if (osd.Contains("Linux"))
```

Comment:
```suggestion
            if (OperatingSystem.IsLinux())
```
`OSDescription` is a free-form string. It is not meant to be used for checks like this.

Reply:
Thanks, I've mimicked what was used below in this file. I'm happy to use the other form.